docs/roman/includes/standard_keywords.inc

```diff
@@ -1,27 +1,25 @@
 
-Standard Keywords
-+++++++++++++++++
-The following table lists the keywords that are *required* to be present in all
-reference files.  The first column gives the keyword name.  The second column 
-gives the roman data model name for each keyword, which is useful when using
-data models in creating and populating a new reference file.  The third column
-gives the equivalent meta tag in ASDF reference file headers, which is the same
-as the name within the data model meta tree (second column).
+Standard ASDF metadata tags
++++++++++++++++++++++++++++
+The following table lists the tags that are *required* to be present
+in all reference files.  The first column gives meta tag in ASDF
+reference file headers, which is the same as the name within the data
+model meta tree (second column).  The second column gives the roman
+data model name for each keyword, which is useful when using data
```

Comment:
May be `The second column gives the fully qualified name of the roman data model attribute.`